scripts/build-src-containers.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+echo "INFO: Build sources containers"
+if [[ -z "${REPODIR}" ]] ; then
+  echo "ERROR: REPODIR Must be set for build src containers"
+  exit 1
+fi
+
+BUILDSH=${REPODIR}/contrail-container-builder/containers/build.sh
+if ! [[ -x "${BUILDSH}" ]] ; then
+  echo "ERROR: build.sh tool from contrail-container-builder is not available in ${REPODIR} or is not executable"
+  exit 1
+fi
+
+PUBLISH_LIST_FILE=${PUBLISH_LIST_FILE:-"src_containers_to_publish"}
```

Comment:
lets use default value which is like
my_file="$(readlink -e "$0")"
my_dir="$(dirname $my_file)"
${my_dir}/src_containers_to_publish

that will allow to run script from any dir